workload_profiling/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/ProfileUtils.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.tool.profiling
```

Comment:
com.nvidia?

Reply:
We are doing this comparison -   `(properties.getOrElse(config.PLUGINS.key, "").contains("com.nvidia.spark.SQLPlugin") ` for which PLUGINS is private  in  `org.apache.spark.internal.config`. So keeping it in org.apache.spark...

Reply:
move it to a trampoline file. ie create some utils class that is in org.apache.spark....  where we can just put specific functions rather then requiring the entire file to be in org.apache.spark